### PR TITLE
Desynchronize vets-api and -worker deployments

### PR DESCRIPTION
**Pull request by patrickvinograd**
Low-tech proposed solution for db:migrate race condition between vets-api and vets-api-worker described in https://github.com/department-of-veterans-affairs/devops/issues/1447

We will likely have a more comprehensive solution when we separate releases (AMI creation) from deployments (instance launch and configuration), so I'm not inclined to majorly refactor vets-api and vets-api-worker deployment to make them an atomic operation.

### CODE REVIEW

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -6,6 +6,12 @@ def notify(message, color='good') {
     }
 }
 
+def delay() {
```
Comment (CyberKoz) [inline Jenkinsfile.deploy]: You should place a comment here stating why this is done. I can see future me finding this and getting indignant as to why someone would put a delay in the build.

**Comment by knkski** [comment]
@patrickvinograd: I didn't think there was a race condition anymore, due to https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/Jenkinsfiles/environment#L81. Are we still seeing problems because of that?

**Comment by patrickvinograd** [comment]
@knkski Your change avoids the race condition when "deploy entire environment" is triggered, but it doesn't do anything for the case of a deploy of vets-api and vets-api-worker triggered by a merge to production in the vets-api repo.

**Comment by patrickvinograd** [comment]
Comment added.

**Pull request merged**